tensorboard/webapp/reloader/reloader_component_test.ts

```diff
@@ -44,6 +67,7 @@ describe('reloader_component', () => {
       declarations: [ReloaderComponent],
     }).compileComponents();
     store = TestBed.get(Store);
+    fakeDocument = TestBed.get(DOCUMENT);
```

Comment:
just wondering, what happens if you change this to `TestBed.get(DOCUMENT) as typeof document`?
I want to avoid `any` where possible.

Reply:
`typeof document` doesn't seem to help.  I can (and have) made `fakeDocument: Document` but it requires some slight grossness to override the visibilityState property. That's ok - I think the grossness is outweighed by the better typing.

Reply:
Acknowledged.